src/test/java/testsuite/regression/StatementRegressionTest.java

```diff
@@ -12707,4 +12707,44 @@ public void testBugIssue450() throws Exception {
             }
         }
     }
+
+    /**
+     * Test fix for Issue #464. Executing a batch statement should ignore any comments.
+     *
```

Comment:
NIT: I'd suggest adding an extra line like
"Verifying that isNonResultSetProducingQuery() properly handles queries that return queryReturnType == QueryReturnType.NONE